vehicle/mkvi/software/sensing/sensing.c

```diff
@@ -0,0 +1,69 @@
+/*
+Header:
+    Code for the Sensing Controller Board located in the nostril motherboard and
+the rear sensing box Author:
+    @author Richard Li
+    @date   03/25/2023
+*/
+
+/*----- Includes -----*/
+#include "sensing.h"
+#include "can_api.h"
+#include "log_uart.h"
+#include "utils/timer.h"
+#include <avr/interrupt.h>
+#include <avr/io.h>
+#include <stdio.h>
+#include <string.h>
+#include <util/delay.h>
+
+void timer1_isr(void) {
+    send_can = true;
+};
+
+int pcint0_callback(int counter_left) {
```

Comment:
If your two counters are global variables, you don't need to take them in as function parameters

Reply:
probably just going to delete them entirely and increment within the struct like you mentioned!